### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

**Comment by fredj** [comment]
+1

File: c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md_tmpl [hunk 1]
```diff
@@ -8,4 +8,4 @@ For this mechanism to work correctly the directive template URL must follow some
 
 In the directive, the template url is written as follow: "templateUrl: {{package}}.baseTemplateUrl + '/<component>/<partial>.html'".
 
-In the main html file, the debug section should contain "{{package}}.templateBaseUrl = '${request.static_url("{{package}}:static-ngeo")}';"
+In the main html file, the debug section should contain "{{package}}.baseTemplateUrl = '${request.static_url("{{package}}:static-ngeo/js/components")}';"
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md_tmpl]: from where does the `components` folder come from ?

Comment (gberaudo) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md_tmpl]: What do you mean? Aren't we organizing the partials by components?

Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md_tmpl]: by components but not in a components folder... ?

Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md_tmpl]: Sorry I'm wrong ...

Comment (gberaudo) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md_tmpl]: OK, I understand what you meant; the component name is prepended in the directive.

**Pull request merged**